api/src/main/scala/com/rasterfoundry/granary/services/StaticService.scala

```diff
@@ -0,0 +1,18 @@
+package com.rasterfoundry.granary.api.services
+
+import cats.effect._
+import org.http4s._
+import org.http4s.dsl.Http4sDsl
+
+class StaticService[F[_]: Sync](blocker: Blocker)(implicit cs: ContextShift[F])
+    extends Http4sDsl[F] {
+
+  val routes: HttpRoutes[F] = HttpRoutes.of[F] {
+
+    case req @ GET -> path =>
+      StaticFile
+        .fromResource(s"/assets$path", blocker, Some(req))
+        .orElse(StaticFile.fromResource("/assets/index.html", blocker, None))
```

Comment:
It _seems_ like it should work with how this is setup, but I think it would be good to have `localhost:8080/` load the web page rather than require `index.html` on the end to make it more user friendly.

Reply:
No joke I had dreams about how to do this last night which uh probably isn't great but oh well

Reply:
550c1d7